### DISCUSSION TITLE
Change Route Name

### DISCUSSION THREAD

**Discussion by anuzpandey** [opcodesio/log-viewer#219] [Q&A] [answered]
Hello.
Is there any way to change to route name `log-viewer.index` to my own route name.

**Comment by arukompas** [comment, marked as answer]
Hey @anuzpandey , why do you need to do that? 🤔  You can create another route with a different name that **redirects** to `log-viewer.index`

**Reply by anuzpandey** [reply]
Well I have a Sidebar generating script which will always require `cms` prefix in the routes. So changing the CMSSidebar array would be cumbersome, so I asked for it if we could do it otherwise.

Yes I did the same thing but like this. 🫢🫢🫢
```php
Route::get('/logs/viewer', static fn() => NULL)->name('log-viewer.index'); // this route are prefixed with cms.
```
